iml-agent/Cargo.toml

```diff
@@ -34,6 +34,7 @@ url = "2.1"
 v_hist = { git = "https://github.com/jgrund/hist", branch = "use-console-instead-of-termion" }
 uuid = { version = "0.7", features = ["v4"] }
 parking_lot = "0.9"
+strfmt = "0.1.6"
```

Comment:
My note:
This is to make `format!` with dynamic format string. It is pretty popular, has all the error constructors public and has no dependencies. Dynamic format is needed to expand `/etc/systemd/system/lamigo-{fs}-{mdt}.service` string.